.travis.yml

```diff
@@ -2,6 +2,8 @@ language: node_js
 matrix:
   include:
   - node_js:
+    - '10'
+    - '8'
     - '6'
```

Comment:
Maybe you can remove 6, its EOL is on the 30th of April.

Reply:
I'm actually thinking of completely dropping travis.

Reply:
@thgreasi sounds good, but maybe as a separate PR? I am ok if you do it in this PR as well TBH.